modal.css

```diff
@@ -0,0 +1,254 @@
+.modal-popup {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  background-color: #191765;
+  background-image: url('./image/Background-image/top-circles.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  box-sizing: border-box;
+  opacity: 0;
+  visibility: hidden;
+  transform: scale(1.1);
+  transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+  display: flex;
+  justify-content: center;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] I see you implemented most of the requested changes, however, there is still an issue with your popup buttons on the desktop version. Please, ensure that there's no popup child element overflowing the container. I know you can fix it, soo go ahead 💪.

- **Your project design:**
![image](https://user-images.githubusercontent.com/80612925/175445470-ada27e88-c01b-423b-96f3-29f46bfcbbe1.png)

**Hint:** `.popup-container {overflow-y: scroll}` and then design your scroll bar or make it invisible. At line **33**